cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -129,6 +136,26 @@ func (spm *KubeStaticPodPathManager) BackupEtcdDir() string {
 	return spm.backupEtcdDir
 }
 
+// CleanupDirs cleans up all temporary directories except those the user has requested to keep around
+func (spm *KubeStaticPodPathManager) CleanupDirs() error {
+	if err := os.RemoveAll(spm.TempManifestDir()); err != nil {
+		return err
+	}
+	if !spm.keepManifestDir {
+		if err := os.RemoveAll(spm.BackupManifestDir()); err != nil {
+			return err
+		}
+	}
+
```

Comment:
minor format consistency things are usually nice to stick in editors to auto-save.  totally non-blocking just an FYI.

Reply:
I do have a lot of autocommit things, just not removing trailing space I guess